crates/bws/README.md

```diff
@@ -18,5 +18,29 @@ Or download a pre-built binary from the [Releases](https://github.com/bitwarden/
 bws --help
 ```
 
+## How to enable shell autocompletions
+
+### Zsh
+
+If completion is not enabled already, you need to enable it first:
+
+```zsh
+echo "autoload -U compinit; compinit" >> ~/.zshrc
+```
+
+Enable autocompletions for the current user:
+
+```zsh
+echo 'source <(/path/to/bws completions zsh)' >> ~/.zshrc
+```
+
+### Bash
+
+Enable autocompletions for the current user:
+
+```zsh
+echo 'source <(/path/to/bws completions bash)' >> ~/.bashrc
+```
```

Comment:
Should this be in the readme or the help docs? The readme is only really visible on crates.io

Reply:
The help docs should probably only be updated after we make a release with these changes merged in I think, otherwise it might be confusing for users of the existing release.